src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/ExplainIT.java

```diff
@@ -61,6 +62,8 @@ public void searchSanity() throws IOException {
         Assert.assertThat(result.replaceAll("\\s+",""), equalTo(expectedOutput.replaceAll("\\s+","")));
     }
 
+    // This test was ignored because group by case function is not supported
+    @Ignore
```

Comment:
If we don't support it , this test case should be failing previously - right?

Reply:
Yes, but I don't think the case should pass, there might be something wrong and need investigating.